### PR TITLE
Remove apache

### PR DESCRIPTION
**Pull request by obino**
Make sure we don't have apache running which may conflict with our normal operations. OCD run is #32, and already passed 2 Hawkeye setups.

### CODE REVIEW

**Comment by obino** [comment]
OCD run with the latest code is https://ocd.appscale.com:8080/job/Hawkeye-Eucalyptus/177/

File: debian/remove-list.awk [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Generate package list from control file.
+
+BEGIN {
+  count = 1
+  flag = 0
+}
+
+# This find all the conflicts, but stop at the first stanza afterward (for
```
Comment (menivaitsi) [inline debian/remove-list.awk]: find -> finds
stop -> stops

File: debian/remove-list.awk [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# example, Descriptions).
+/^(Conflicts):/,/^Description:/ {
+    if (gsub("^Conflicts:", "")) { flag = 1 }
+    if ($0 !~ /^[ \t]/) { flag = 0 }    # stop at the first stanza
```
Comment (menivaitsi) [inline debian/remove-list.awk]: stop- > Stop
Also put a '.'.

File: debian/remove-list.awk [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    if (flag == 0) next
+    split($0, list, ",");
+    for (i in list) {
+        if (length(list[i]) > 0) {
+            gsub("\(.+\)", "", list[i])
+            # let's make sure we are not adding packages that we should
```
Comment (menivaitsi) [inline debian/remove-list.awk]: let's -> Let's

File: debian/remove-list.awk [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            # create here (ie appscale-*)
```
Comment (menivaitsi) [inline debian/remove-list.awk]: Put a '.'.

**Comment by menivaitsi** [comment]
Good work! :)

**Pull request merged**